### ISSUE TITLE
Alfred 3 Support?

### ISSUE THREAD

**Issue by VxJasonxV** [jason0x43/alfred-nest#4]
Does this plugin support Alfred 3? I hadn't seen anything about AppleScript changes.

I've never received any kind of login prompt, and subsequently any commands (`nest nest`, `nest status`) never return any information. Even `nest log` closes Alfred upon pressing enter but doesn't invoke anything.

**Comment by fdstevex** [comment]
It's working for me with Alfred 3.

**Comment by VxJasonxV** [comment]
But did you configure it (auth to Nest) before upgrading to Alfred 3?

**Comment by megalithic** [comment]
@VxJasonxV man, i feel so dumb;  mind sharing details on auth'ing with Nest for Alfred3 support? Thanks tons!

**Comment by VxJasonxV** [comment]
Hi Megalithic. Something strange happened here. Either I downloaded a particularly old (Pre-Go) version of the plugin, or whatever plugin I had installed wasn't actually jason0x43's in the first place. Hard to say since I deleted the previous one in order to debug and didn't think to retain it (whoops).

Updating to the latest release via the link in the README allowed me to authorize and get this working.

Alfred 3.1 pre-release, macOS Sierra, all working. Fun!
